plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/support/SupportZip.js

```diff
@@ -125,8 +125,12 @@ Ext.define('NX.coreui.view.support.SupportZip', {
           text: NX.I18n.get('Support_SupportZip_Create_Button'),
           formBind: true,
           glyph: 'xf019@FontAwesome' /* fa-download */,
-          action: 'save',
+          action: 'add',
```

Comment:
‘add’ is used by all the drilldown features when adding a new object. The behavior here is the same (since it should be possible to create a support zip when the form hasn’t been dirtied), but agree that the name is a bit misleading.

Rename all of the ‘add’ instances to ‘submit’?